source/Octodiff/Core/BinaryDeltaReader.cs

```diff
@@ -90,9 +92,10 @@ void EnsureMetadata()
                     long soFar = 0;
                     while (soFar < length)
                     {
-                        var bytes = reader.ReadBytes((int) Math.Min(length - soFar, 1024*1024*4));
-                        soFar += bytes.Length;
-                        writeData(bytes);
+                        var chunkLength = (int)Math.Min(length - soFar, buffer.Length);
```

Comment:
Nit: Use the `const` here rather than evaluating the buffer length.

Reply:
Thanks Adam, this has been addressed [here](https://github.com/OctopusDeploy/Octodiff/pull/39/commits/5b886de05dd2fa0ca1118cdefbc894ed6652e693).